packages/ng/forms/text-input/text-input.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="textField">
+<div class="textField" [class.mod-alignRight]="alignRight">
```

Comment:
I’m wondering if specifying what we’re aligning right would be a good thing. What if we want to align the help text to the right of the container later ?

Maybe `valueAlignRight`?